src/Symfony/Component/Form/FormRegistry.php

```diff
@@ -120,7 +120,7 @@ private function resolveAndAddType(FormTypeInterface $type)
         $hasCustomName = $name !== $fqcn;
 
         if ($parentType instanceof FormTypeInterface) {
-            @trigger_error('Returning a FormTypeInterface from FormTypeInterface::getParent() is deprecated since version 2.8 and will be removed in 3.0.', E_USER_DEPRECATED);
+            @trigger_error(sprintf('Returning a FormTypeInterface from %s::getParent() is deprecated since version 2.8 and will be removed in 3.0.', $fqcn), E_USER_DEPRECATED);
```

Comment:
You could add `Return the fully-qualified type class name instead.` there as well.